src/import/FSHImporter.ts

```diff
@@ -1984,7 +2093,21 @@ export class FSHImporter extends FSHVisitor {
   }
 
   private extractStartStop(ctx: ParserRuleContext): TextLocation {
-    if (ctx instanceof TerminalNode) {
+    if (pc.isStarContext(ctx)) {
+      const location = {
+        startLine: ctx.STAR().symbol.line + 1,
+        startColumn: this.getStarContextStartColumn(ctx),
+        endLine: ctx.stop.line,
+        endColumn: ctx.stop.stop - ctx.stop.start + ctx.stop.column + 1
+      };
+      if (!pc.containsPathContext(ctx) && location.startColumn - this.baseIndent > 0) {
+        logger.error(
+          'A rule that does not use a path cannot be indented to indicate context. The rule will be processed as if it is not indented.',
+          location
```

Comment:
I think this is the one remaining log that needs to be updated. Besides that, everything looks good!

Reply:
Thank you for your diligence in the face of my whoopsies, https://github.com/FHIR/sushi/pull/797/commits/fca5d4621d1abe80b66908dabe5a30358980d2be fixes this.

Reply:
Thank you to the regression script for feigning diligence for me.